manager/assets/modext/core/modx.component.js

```diff
@@ -104,7 +104,7 @@ Ext.extend(MODx.Component,Ext.Component,{
         return true;
     }
 });
-Ext.reg('modx-component',MODx.Component);
```

Comment:
that is from when I was trying to figure out how to change the order of where the action buttons where on the page, but I do't think all those changes to modx.component.js are needed. I'll try removing some of them